awesomplete.js

```diff
@@ -109,6 +109,14 @@ var _ = self.Awesomplete = function (input, o) {
 		inside: this.container
 	});
 	
+	this.status = $.create("span", {
+		hiddena11y: "",
```

Comment:
The way that a aria-live region works is that it announces its content the moment this content has changed. The list items in the dropdown are generated once and only updated when a new search result is generated. Reading them out then doesn't help. We need to announce the currently selected item so the user can decide to select it. So we can either set focus to the selected item (perfectly valid way to do this but it would change the way awesomeplete works at the moment) or we create 1 hidden live-region and have it read whatever we currently select by pushing the messages there. Sorry Lea there's no other option.